liblangutil/Token.h

```diff
@@ -367,6 +367,8 @@ class ElementaryTypeNameToken
 		solAssert(name.size() >= 3, "Token name size should be greater than 3. Should not reach here.");
 		if (m_token == Token::FixedMxN || m_token == Token::UFixedMxN)
 			return name.substr(0, name.size() - 3) + std::to_string(m_firstNumber) + "x" + std::to_string(m_secondNumber);
+		else if (m_token == Token::Address)
```

Comment:
What exactly is this change?

Reply:
Otherwise the type would be called `addres160`  which is wrong

Reply:
in the AST output.

Reply:
Could it rather be that the import incorrectly sets ``_firstNumber`` to 160 instead of 0 for ``address``? I agree with @axic, this actually looks fishy...

Reply:
I think it's a consequence of this 060 addition: https://github.com/ethereum/solidity/blob/develop_060/libsolidity/parsing/Parser.cpp#L1681

Reply:
That's commit 5cfe0b767
I am currently testing how the AST output of that commit looks

Reply:
And maybe the correct fix would be to remove the 160 in that line, yes. I'll test this.

Reply:
Fixed in #7923.